pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -380,11 +380,11 @@ func (r *Builder) mapFirmware(vm *model.Workload, cluster *model.Cluster, object
 	}
 	switch biosType {
 	case Q35Ovmf:
-		smmEnabled := true
-		features.SMM = &cnv.FeatureState{
-			Enabled: &smmEnabled,
-		}
-		firmware.Bootloader = &cnv.Bootloader{EFI: &cnv.EFI{}}
+		secureBootEnabled := false
+		firmware.Bootloader = &cnv.Bootloader{
+			EFI: &cnv.EFI{
+				SecureBoot: &secureBootEnabled,
+			}}
```

Comment:
the changes look good but how about documenting (in the code) why we do that?

Reply:
hmm this part is actually self-explained, I meant after adding this handling to the other 'bios-type' that represents uefi+secure boot